packages/react/src/components/Tabs/Tabs-test.js

```diff
@@ -220,6 +222,17 @@ describe('Tabs', () => {
           expect(wrapper.state().selected).toEqual(1);
         });
 
+        it('updates selected state when pressing the End key', () => {
+          firstTab.simulate('keydown', { which: endKey });
```

Comment:
One thing you can do is:

```js
import { End } from '../../../internal/keyboard';

// ...

firstTab.simulate('keydown', End);
```

Just so you don't have to manage these in the tests themselves 👍

Reply:
yeah was just following the existing test format, just refactored to use the objects from `keys.js`